XamarinCommunityToolkit.UnitTests/Converters/StringToIntConverter_tests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Globalization;
+using Xamarin.CommunityToolkit.Converters;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests.Converters
+{
+	public class StringToIntConverter_tests
+	{
+		[Theory]
+		[InlineData("2", 2)]
+		[InlineData("3", 3)]
+		[InlineData("255", 255)]
+		[InlineData("99191", 99191)]
```

Comment:
Can we also maybe add some test cases here that we expect to fail?

Reply:
Please add test cases where the exception is expected